tests/fixtures/snaps/basic-hooks/meta/snap.yaml

```diff
@@ -0,0 +1,2 @@
+name: basic-hooks
+version: 1.0
```

Comment:
Only if they require plugs. Otherwise they're implicit if they're included in the snap.